### PR TITLE
Tx generator: pure availableFunds.

### PR DESCRIPTION
**Pull request by denisshevchenko**
After #359 was done, we saw that `availableFunds` should not be `TVar` actually. Now we have to turn it from the mutable value to a pure one.

This PR fixes `log-config-explorer.yaml` for local cluster as well.

### CODE REVIEW

File: cardano-node/src/Cardano/CLI/Tx/Generation.hs [hunk 1]
```diff
@@ -816,16 +770,22 @@ createMoreFundCoins llTracer
...
-  liftIO $ forM_ splittingTxs $ \(tx, txDetailsList) -> do
+                                         (genesisTxDetails, sourceKey)
+                                         numSplittingTxOuts
+                                         numOutsPerSplittingTx
+                                         42
+                                         txOut
+                                         []
+  -- Submit all splitting transactions sequentially.
+  liftIO $ forM_ splittingTxs $ \(tx, _) ->
     submitTx socketFp pInfoConfig (CoreId 0) tx llTracer
```
Comment (ksaric) [inline cardano-node/src/Cardano/CLI/Tx/Generation.hs]: So all the (splitting) transactions go into a single node (`CoreId 0`)?

Comment (denisshevchenko) [reply cardano-node/src/Cardano/CLI/Tx/Generation.hs]: Yes.

File: cardano-node/src/Cardano/CLI/Tx/Generation.hs [hunk 2]
```diff
@@ -816,16 +770,22 @@ createMoreFundCoins llTracer
...
-    -- Update available fundValueStatus to reuse the numSplittingTxOuts TxOuts.
-    forM_ txDetailsList (addToAvailableFunds availableFunds)
+
+  -- Re-create availableFunds with information about all splitting transactions
+  -- (it will be used for main transactions).
+  right . Set.fromList $ map (\txDetails -> FundValueStatus { status    = Unknown
```
[the remark this answers is not in the document]
Comment (denisshevchenko) [reply cardano-node/src/Cardano/CLI/Tx/Generation.hs]: Done.

File: cardano-node/src/Cardano/CLI/Tx/Generation.hs (hunk 1 above)
Comment (CodiePP) [reply cardano-node/src/Cardano/CLI/Tx/Generation.hs]: this is not subject to benchmarking; it's OK

**Pull request merged**